spec/stack_factory_spec.rb

```diff
@@ -23,4 +24,23 @@
     expect(stack.parameter_defaults[:desired_capacity]).to eq 2
   end
 
+  it 'manages volatile parameters' do
+    stack = described_class.build(id: :bar, parameter_context: OpenStruct.new) do
+      name "something"
+      capabilities [:iam]
```

Comment:
this looks like a let() or a def build_stack.  imo, it's best to keep the actual tests small and easy to grok.

Reply:
This definition is used only by this spec, so I will leave it in here.